### ISSUE TITLE
Broken in pytest == 6.0.0

### ISSUE THREAD

**Issue by Focus** [klen/pylama#187]
The warning message in #177 is now an error in pytest version 6.0.0
```
E   pytest.PytestDeprecationWarning: Direct construction of PylamaItem has been deprecated, please use PylamaItem.from_parent.
E   See https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent for more details.
```

**Comment by ubaumann** [comment]
@klen Woud you mind checking out my PR and give some feedback if I need to change something? The test is failing but I created an other PR who fix it but it's maybe not the most beautiful fix.

**Comment by klen** [comment]
Thank you for the feedback. The issue has been fixed in the latest version (8.0.4).
